### DISCUSSION TITLE
Extension Support?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1333] [Q&A]
Hey there, I recently stumbled upon Bromite and decided to try it out for myself and I am rather impressed. I do have a question though and feel free to close/delete this discussion but are there any plans to support extensions?

**Comment by csagan5** [comment]
See https://github.com/bromite/bromite/blob/master/FAQ.md#does-bromite-support-extensions

**Reply by chromer030** [reply]
There was an on-going [project](https://github.com/ungoogled-software/ungoogled-chromium-android) which has added extension support for a while but because of lots of downstream works supporting the feature [discontinued](https://github.com/ungoogled-software/ungoogled-chromium-android/issues/78).

**Reply by ghost** [reply]
Man, that is so upsetting to see :( I guess only time will tell if Extension Support will come.

**Reply by hbarsaiyan** [reply]
@MrTeeXD have a look at https://github.com/kiwibrowser/src.next if u want extension support.

**Reply by ghost** [reply]
How do I know that Kiwi is privacy friendly? I love my privacy

**Reply by hbarsaiyan** [reply]
It doesn't really have any privacy features (apart from disabling FLOC and removing amp links) AFAIK. That is why I say u can look at it if u want extension support otherwise u r better off on bromite.

**Reply by chromer030** [reply]
Extension support on kiwi has lots of bugs, because upstream doesn't support this feature and most of extensions has not been optimized for smartphones

**Reply by Dastardly-Entrench** [reply]
> How do I know that Kiwi is privacy friendly? I love my privacy

It's not, see https://github.com/Tobi823/ffupdater/issues/35

Also read the business model under kiwi src GitHub page.
